### PR TITLE
Settings branch

### PR DESCRIPTION
**Pull request by padawin**

### CODE REVIEW

File: js/bass.js [hunk 1]
```diff
@@ -22,12 +22,12 @@ angular.module('bassPracticeApp', [])
...
 		}
 
-		note = FretBoardFactory.getRandomNote(SystemsFactory.getSelected());
+		note = SystemsFactory.getRandomNote();
 		canPlay = true;
 	}
 
 	function start(initTurns) {
-		turn = totalTurns = initTurns;
+		turn = totalTurns = parseInt(initTurns) || 5;
```
Comment (brettminnie) [inline js/bass.js]: should you not default here to `getDefaultTurnsNumber()`?

File: js/bass.js [hunk 2]
```diff
@@ -113,9 +118,75 @@ angular.module('bassPracticeApp', [])
  * Romance (Do Ré Mi Fa Sol La Si) and All (both previous combined).
  */
 .factory('SystemsFactory', function() {
-	var systems;
+	var systems, chords, selected, scaleLength;
+
+	chords = [
```
Comment (brettminnie) [inline js/bass.js]: correctly these are notes or intervals

File: js/bass.js [hunk 3]
```diff
@@ -113,9 +118,75 @@ angular.module('bassPracticeApp', [])
...
+		// English
+		'C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'B',
+		// Romance
+		'Do', 'Do#', 'Re', 'Re#', 'Mi', 'Fa', 'Fa#', 'Sol', 'Sol#', 'La', 'La#', 'Si',
+	];
+
+	scaleLength = 12;
```
Comment (brettminnie) [inline js/bass.js]: fingerboardLength perhaps, scale length is a technical term in fretting instrument construction

File: js/bass.js [hunk 4]
```diff
@@ -113,9 +118,75 @@ angular.module('bassPracticeApp', [])
...
+	}
+
+	/**
+	 * Returns a note in a given system located on a given string on
+	 * a given fret
+	 *
+	 * @returns string|array String if the system is english/romance,
+	 * 		array if the system is all
+	 */
+	function _getChord(system, openString, fret) {
```
Comment (brettminnie) [inline js/bass.js]: getNote, your comment even says this :)

A chord is a grouping of 2 or more notes played simultaneously

File: js/bass.js (hunk 1 above)
Comment (padawin) [inline js/bass.js]: You are very right.

**Pull request merged**